courses/free-courses-te.md

```diff
@@ -1,39 +1,222 @@
+
+
+### Bootstrap
+
+* [BOOTSTRAP INTRODUCTION IN TELUGU](https://www.youtube.com/watch?v=1MrR6eTQYgI) - TeluguTechSteps
+* [Bootstrap Tutorials in Telugu by Kotha Abhishek](https://www.youtube.com/playlist?list=PLv_sM9ZH4RUU9Yopth1p1DbEu1ebr1vTb) - Chintu Tutorials
 
 
 ### C
 
-* [C Language Tutorials](https://www.youtube.com/playlist?list=PLC2mgeYbYNm-n8Iz-_3MuNsJFzr0UlGUu) - Telugu Computer World (YouTube)
+* [C Language in Telugu - Complete Tutorial in 12 Hours](https://www.youtube.com/watch?v=HdvRHC5TiwE) - Telugu Computer World
+* [C Language Tutorial](https://www.youtube.com/playlist?list=PL3KKfF5A0sSKZutcrUiTCQDX0hn7Tw61E) - Telugu Scit Tutorials
+
+
+### C++
+
+* [C++ Complete Tutorial](https://www.youtube.com/watch?v=uZBXKmQH5u8) - Telugu Computer World
+* [C++ Programming in Telugu - Complete Tutorial in 14 Hours](https://www.youtube.com/playlist?list=PLC2mgeYbYNm9keJjsA95jKa4EUVLd7mQP) - Telugu Computer World
+
+
+### Cloud Computing
+
+* [Introduction to Cloud Computing in Telugu | Cloud Computing Tutorial in Telugu | Edureka Telugu](https://www.youtube.com/watch?v=hcaROQRJM6k) - edureka! Telugu
+* [Cloud Computing](https://www.youtube.com/playlist?list=PL35ft-0sAlPhkcplnefpnc7U4BzLTh_Uh) - Cloud Computing in Telugu
+
+
+### Compiler Design
+
+* [Compiler Design Telugu Series](https://www.youtube.com/playlist?list=PLXs97PqiPGv_KtMEdRHNxrjRWuYIIxRtK) - Jahnavi Raghava Singh
+
+
+### CSS
+
+* [CSS Tutorial for Beginners in Telugu | Best CSS tutorial for beginners | CSS3 Tutorial | TechEnlgiht](https://www.youtube.com/watch?v=z7_gt7x6XAM) - TECH ENLIGHT
+* [CSS Tutorials in telugu || CSS in Telugu by Kotha Abhishek](https://www.youtube.com/playlist?list=PLv_sM9ZH4RUVjmxTl5PysFSxJ6VQbdnRc) - Chintu Tutorials
+
+
+### Database Management Systems
+
+* [Data Base Management System In Telugu in 7hrs | DBMS in telugu | MySql Full Course](https://www.youtube.com/watch?v=nVgLiJOI2U8) - Believer 01
+* [DBMS Tutorial](https://www.youtube.com/playlist?list=PL3KKfF5A0sSLnIMTfr7bBw_wRW2vCm3T6) - Telugu Scit Tutorials
+
+
+### Data Science
+
+* [Data Science Course For Beginners in Telugu](https://www.youtube.com/watch?v=WKHlx--15_I) - Python Life
+* [Maths for DS](https://www.youtube.com/watch?v=8Y982AW4KuY) - Vamsi Bhavani
+
+
+### Data Structures
 
+* [Data Structures in Telugu in 7hrs | Full Course | Learn Data Structures](https://www.youtube.com/watch?v=pm_ugbO2FlY) - Believer 01
+* [Data Structures](https://www.youtube.com/playlist?list=PLJSrGkRNEDAgmq4kKkPuh8aFJs-zxVbWK) - Lab Mug
 
-### <a id="cpp"></a>C++
 
-* [C++ Language Tutorials](https://www.youtube.com/playlist?list=PLC2mgeYbYNm9keJjsA95jKa4EUVLd7mQP) - Telugu Computer World (YouTube)
+### Deep Learning
+
+* [Deep learning in Telugu - 1 | Deep learning tutorials in Telugu | Jp NaN Deep learning](https://www.youtube.com/watch?v=nya9vUaXAGY) - Jp NaN
```

Comment:
This is only for 19 mins. We dont allow tutorials < 1hr in length. Please remove